lib/net/pool.js

```diff
@@ -3772,12 +4086,17 @@ class PoolOptions {
 
     if (options.spv != null) {
       assert(typeof options.spv === 'boolean');
-      assert(options.spv === this.chain.options.spv);
+      // assert(options.spv === this.chain.options.spv);
       this.spv = options.spv;
     } else {
       this.spv = this.chain.options.spv;
     }
 
+    if (options.neutrino != null) {
```

Comment:
assert for compact blocks to be turned off

Reply:
Okay